scripts/libtss2_build.py

```diff
@@ -3,8 +3,29 @@
 ffibuilder = FFI()
 
 import os
+import pkgconfig
+import re
 import sys
 
+
+def get_include_paths(library_names):
+    if not isinstance(library_names, list):
+        library_names = [library_names]
+
+    # find search paths for libraries and header via pkg-config
+    header_dirs = set()
+
+    # include locally installed packages on common distros
+    os.environ["PKG_CONFIG_PATH"] = "/usr/lib/local"
```

Comment:
I don't think we want to clobber PKG_CONFIG_PATH as it could be set by the user. I also don't think we want to add any differing behavior from autotools. In those cases you have to export PKG_CONFIG_PATH or change the config.

Reply:
You're right. Removing this.

At some point, we might want to point out in the build instructions that the user can point to a custom TSS installation via the `PKG_CONFIG_PATH`.